core/parameters/res/runtime_configs/parameters.snap

```diff
@@ -160,7 +160,7 @@ max_length_method_name                                   256
 max_arguments_length                               4_194_304
 max_length_returned_data                           4_194_304
 max_contract_size                                  4_194_304
-max_transaction_size                               4_194_304
```

Comment:
Not necessarily, as the size limit for receipts is still 4MiB, so I think that a contract could potentially deploy another 4MiB contract.

Reply:
But still it will make it impossible to deploy a large contract using the most common case of deploy action in a single transaction right? Just to be clear I'm not against it, just making sure that you considered this consequence.

Reply:
> But still it will make it impossible to deploy a large contract using the most common case of deploy action in a single transaction right? 

Yes, it's a breaking change :/

I'm currently focused on limiting transactions, so this PR only adds a limit for transactions, but in the future that might be extended to receipts as well. (possibly for https://github.com/near/nearcore/issues/11295)